### PR TITLE
Dice

### PR DESCRIPTION
**Pull request by Gonaline**

### CODE REVIEW

**Review by n-faycel — DISMISSED** [review]
ok

File: backend/package.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
     "cors": "^2.8.5",
     "dotenv": "^10.0.0",
     "express": "^4.17.2",
-    "mysql2": "^2.3.3"
+    "mysql2": "^2.3.3",
+    "template-fullstack": "file:.."
```
Comment (LuigiMenez) [inline backend/package.json]: We really need to find a solution for this

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
no template fullstack

**Review by LoicBrassart — APPROVED** [review]
No more Evil

**Pull request merged**